.github/workflows/run-test-cases.yml

```diff
@@ -266,10 +276,10 @@ jobs:
       - if: (startsWith(github.event_name, 'pull_request')) && (startsWith(matrix.kube.runtime, 'Kubernetes'))
         name: Import local agent and controller to Kubernetes
         run: |
-          sudo docker load --input agent.tar
-          sudo docker load --input controller.tar
-          sudo docker load --input webhook-configuration.tar
-          sudo docker image ls
+          #sudo docker load --input agent.tar
```

Comment:
Instead of commenting out, i think these commands are fine to remove now that we are importing containers with `ctr` for containerd instead

Reply:
Yes I hesitated but you're right it's useless now, fixed.